pytorch_lightning/plugins/precision/apex_amp.py

```diff
@@ -75,15 +75,7 @@ def backward(
         closure_loss = scaled_loss.__enter__()
 
         # do backward pass
-        # TODO: not entirely sure, why we need this
-        if model is not None and isinstance(model, LightningModule):
-            model.backward(closure_loss, optimizer, opt_idx, **kwargs)
```

Comment:
This is a BC change right ?

Reply:
Was just trying things. This PR is a draft - will revert this